src/infrastructure/data/index.js

```diff
@@ -423,7 +423,7 @@ const getInvitationServices = async (iid) => {
 };
 
 const getInvitationService = async (iid, sid, oid) => {
-  const entities = await invitationServices.find({
+  const entities = await invitationServices.findOne({
```

Comment:
for more than one invittaion . we need sort by date desc. So it picks up latest invitation